presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/SliderClusterFulfiller.groovy

```diff
@@ -38,7 +38,9 @@ public class SliderClusterFulfiller
   public static final String PACKAGE_NAME = 'PRESTO'
 
   private static final Path SLIDER_BINARY = Paths.get('target/package/slider-assembly-0.80.0-incubating-all.zip')
-  private static final Path PRESTO_PACKAGE = Paths.get('target/package/presto-yarn-package-1.0.0-SNAPSHOT.zip')
+  @Inject
+  @Named("tests.resources.path")
```

Comment:
tests.resources.path suggests something different than presto-app package. Maybe name it `tests.app.package.path'